snippets/graph_neighbourhood.py

```diff
@@ -0,0 +1,68 @@
+""""""
+
+from itertools import starmap
+from operator import le, lt, and_
+from pathlib import Path
+from os import path
+from sys import argv, stdout
+from argparse import ArgumentParser
+
+from data import load_graph
+
+
+def neighbourhood(coord: tuple[int, int], borders) -> None:
```

Comment:
Including a docstring for the function renders readability to the code. You can consider using the [Numpy-doc standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) for writing a docstring.

Good to see that the type is annotated for the first function argument; you could do the same for the second argument 'borders' as well.